products/opensuse/main.pm

```diff
@@ -654,19 +665,50 @@ sub load_extra_tests () {
     return 0;
 }
 
+sub load_otherDE_tests() {
```

Comment:
can you rename this to "load_other_de_tests"? I don't mind the upper case but in your local verification run I saw that it says "otherde_…" and the "otherde" looks weird